docs/source/api/v4/servers_id_deliveryservices.rst

```diff
@@ -21,7 +21,7 @@
 
 ``GET``
 =======
-Retrieves all :term:`Delivery Services` assigned to a specific server.
+Retrieves all :term:`Delivery Services` assigned to a specific server either directly or inherited from topology.
```

Comment:
This retroactively modifies a released API? Can we do that? Is this considered a bug, or do people rely on the current behavior and assume this will omit Topology-assigned servers?

Reply:
Addressed directly to reviewer.